### DISCUSSION TITLE
Spacy for Commercial Use

### DISCUSSION THREAD

**Discussion by saravanan0206** [explosion/spaCy#9104] [Help: Best practices]
Hi, 
I would like to use Spacy for Commercial products. But a vulnerability scan report says that the below packages which are used by Spacy are on License Risk (medium). 

        chardet - GNU Lesser General Public License v2.1 only
        certifi - Mozilla Public License 2.0
        tqdm - MIT License AND Mozilla Public License 2.0

Please share your suggestion

**Comment by polm** [comment]
We are not legal professionals and cannot give legal advice, and if you are concerned you should hire a professional. 

Our understanding is that using unmodified LGPL or MPL libraries in Python projects is not an issue in commercial software.
